Makefile

```diff
@@ -0,0 +1,36 @@
+BUILD_ARCH := $(shell uname -m)
+RELEASE_NAME := "dragonfly-${BUILD_ARCH}"
+
+HELIO_RELEASE := $(if $(HELIO_RELEASE),y,)
+HELIO_RELEASE_FLAGS = -DHELIO_RELEASE_FLAGS="-flto"
+HELIO_USE_STATIC_LIBS = ON
+HELIO_OPENSSL_USE_STATIC_LIBS = ON
+HELIO_ENABLE_GIT_VERSION = ON
+HELIO_WITH_UNWIND = OFF
+
+HELIO_FLAGS = $(if $(HELIO_RELEASE),-release $(HELIO_RELEASE_FLAGS),) \
+              -DBoost_USE_STATIC_LIBS=$(HELIO_USE_STATIC_LIBS) \
+              -DOPENSSL_USE_STATIC_LIBS=$(HELIO_OPENSSL_USE_STATIC_LIBS) \
+              -DENABLE_GIT_VERSION=$(HELIO_ENABLE_GIT_VERSION) \
+              -DWITH_UNWIND=$(HELIO_WITH_UNWIND) \
+
+.PHONY: default
+
+configure:
+	./helio/blaze.sh $(HELIO_FLAGS)
+
+build:
+	cd build-opt; \
```

Comment:
Unless I am mistaken, this makefile is intended to build only the release version of dragonfly. 
From this code it seems like it because you cd to build-opt which is created only for release.

if you intend to extend the makefile functionality to debug variant as well, then it support "cd build-dbg".

Reply:
if you do not want to support debug builds then this `HELIO_RELEASE` logic is over-complication - you can just hardcode these flags into HELIO_FLAGS without requiring a caller to pass a special flag.

Reply:
I guess we should support debug builds as well, while we're already in here. Might make it easier down the road.